builtins.go

```diff
@@ -1211,11 +1211,6 @@ func jsonEncode(v interface{}) (string, error) {
 // For backwards compatibility reasons, we are manually marshalling to json so we can control formatting
 // In the future, it might be apt to use a library [pretty-printing] function
 func builtinManifestJSONEx(i *interpreter, arguments []value) (value, error) {
-	obj, err := i.getObject(arguments[0])
```

Comment:
Nit: Let's keep it named. `value := arguments[0]`. This way it's more clear what the args are.

Reply:
calling it `value` will shadow the type with the same name, which breaks the funcs that declare parameters of that type inside of the scope of `buildinManifestJSONEx`.
How about `val`?

Reply:
Good point. `val` sounds good.